packages/dataviz/src/utils/chart-settings/metric-chart-settings.ts

```diff
@@ -0,0 +1,79 @@
+import { VisualizationOptions } from './types';
+import { MetricChartSettings } from '@keen.io/charts';
+import { Analysis } from '@keen.io/query';
+import { INTERVAL_UNITS } from '../../../../charts-utils/src/constants';
```

Comment:
please export this const from utils

`import { INTERVAL_UNITS } from '@keen.io/charts-utils';`